lib/utils/dataflow/src/index.js

```diff
@@ -523,6 +580,34 @@ const mapper = (mapfn, opt) => {
       };
     }));
 
+  // Retrieve error docs
+  if(opt.error && opt.error.get)
+    routes.get(opt.error.get, throttle(function *(req) {
+      const ts = map(filter(
+        pairs(req.params), (p) => /^t/.test(p[0])), (p) => p[1]).join('/');
+
+      debug('Retrieving error docs from beginning of the month to %s', ts);
```

Comment:
This was the simplest implementation of get that I thought of at the moment.

Operator would pass a time, and dataflow would returns all errors in the database from the beginning of the month(based on the time passed) to the time that the operator passed.

Should I returns all docs? I didn't put much thought into this because I am not sure with the use case, and would like to get actual comments from users and enhance.